doc/content/manuals/include/materials/isoil-user.md

```diff
@@ -1,13 +1,13 @@
 ### The I_Soil System
 
 The I-soil material model is a nonlinear hysteretic soil model that is based on the distributed
-element models developed by [citet:iwan1967on] and [citet:chiang1994anew]. In 1-D, this model takes
+element models developed by [!citet](iwan1967on) and [!citet](chiang1994anew). In 1-D, this model takes
 the backbone stress-strain curve and divides it into a set of elastic-perfectly plastic curves. The
 total stress then is the sum of the stresses from the individual elastic-perfectly plastic curves.
 
 The three dimensional generalization of this model is achieved using von-Mises failure criteria for
 each elastic-perfectly plastic curve resulting in an invariant yield surfaces in three-dimensional
-stress space like in [fig:yieldsurface] (after [citet:chiang1994anew]).
+stress space like in [fig:yieldsurface] (after [!citet](chiang1994anew)).
```

Comment:
Same here.

Reply:
```suggestion
stress space like in [fig:yieldsurface]
(after [!citet](chiang1994anew)).
```

Reply:
I am not able to commit those suggestions. Can you please commit? I can merge then.